example.py

```diff
@@ -2,5 +2,7 @@ def add(a,b):
   return a + b
 
 def subtract(a,b):
-  return a * b
+  return a - b
 
+def multiply(a,b):
```

Comment:
I like more `product` 
```suggestion
def product(a,b):
```